### ISSUE TITLE
During initial installation there seems to be a broken partial

### ISSUE THREAD

**Issue by kdamianakis** [google/docsy-example#91]
During initial installation there seems to be a broken partial - could you please advise?

"section" failed: execute of template failed: template: blog/section.print.html:2:3: executing "main" at <partial "print/render" .>: error calling partial: execute of template failed: template: partials/print/render.html:10:4: executing "recurse-toc" at <partial "print/toc-li.html" (dict "sid" $sid "Page" .)>: error calling partial: " .. /hugo/docsy-example/themes/docsy/layouts/partials/print/toc-li.html:3:5": execute of template failed: template: partials/print/toc-li.html:3:5: executing "partials/print/toc-li.html" at <partial $tpl .>: error calling partial: partial "/print/toc-li-blog.html" not found

Please let me know if any more information is needed.

**Comment by gwatts** [comment]
Hi @kdamianakis  - I tried to reproduce this just now:

```
$ cd /tmp
gareth@1573HTDF tmp $ git clone --recurse-submodules --depth 1 https://github.com/google/docsy-example.git
Cloning into 'docsy-example'...
remote: Enumerating objects: 160, done.
... (clone details omitted)
$ cd docsy-example/
gareth@1573HTDF docsy-example (master=) $ hugo server
Start building sites …

                   | EN | FA | NO
-------------------+----+----+-----
  Pages            | 47 | 17 | 76
  Paginator pages  |  0 |  0 |  0
  Non-page files   |  3 |  3 |  1
  Static files     | 38 | 38 | 38
  Processed images |  8 |  7 |  2
  Aliases          |  4 |  2 |  0
  Sitemaps         |  2 |  1 |  1
  Cleaned          |  0 |  0 |  0

Built in 1333 ms
Watching for changes in /tmp/docsy-example/{assets,content,layouts,package.json,themes}
Watching for config changes in /tmp/docsy-example/config.toml, /tmp/docsy-example/themes/docsy/config.toml
Environment: "development"
Serving pages from memory
Running in Fast Render Mode. For full rebuilds on change: hugo server --disableFastRender
Web Server is available at //localhost:1313/ (bind address 127.0.0.1)
Press Ctrl+C to stop
```

so seems to be working (for me at least).. can you check if your clone has the `themes/docsy/print/toc-li-blog.html` file?

Also which version of Hugo are you using?  Did you make any other changes, or is this also a completely clean checkout that you're testing against?

**Comment by kdamianakis** [comment]
Thank you for this.

No I have made no changes.

I am currently using the extended version (Hugo Static Site Generator v0.74.3/extended darwin/amd64 BuildDate: unknown) although I had the same error with the static version. I have just tried it again with same results. For some reason I do not  have/it does not create a print folder.

063 macbook-pro-/ .. /hugo/docsy-example/themes/docsy->ls
CONTRIBUTING.md   README.md         config.toml       images            netlify.toml      static            userguide
LICENSE           assets            i18n              layouts           postcss.config.js theme.toml

**Comment by gwatts** [comment]
Sounds quite odd!  What command line did you use to checkout the example?  It should of pulled in a version of docsy that includes that print partial

if you cd into the `docsy-example` example and run `git submodule`, what does it show?  Should return the following:

```
areth@1573HTDF docsy-example (master=) $ git submodule
 62cc3edbea013465179e1ea641d0d0da4fdba6f0 themes/docsy (62cc3ed)
```

If you cd into `themes/docsy` does it show a clean checkout?

```
$ cd themes/docsy/
gareth@1573HTDF docsy ((62cc3ed...)) $ git status
HEAD detached at 62cc3ed
nothing to commit, working tree clean
```

**Comment by kdamianakis** [comment]
Yes I thought it was strange - I could not recreate your (good) results..

- cd into the docsy-example example and run git submodule shows the following (I think that is the correct git hash)
 62cc3edbea013465179e1ea641d0d0da4fdba6f0 themes/docsy (remotes/origin/LisaFC-patch-2-22-g62cc3ed)

- regarding git status on docsy-example/themes/docsy->git status
HEAD detached at 62cc3ed
nothing to commit, working tree clean

I like the theme (and would like to use it for an internal app documentation) but for some reason I cannot get a clean build and do not know enought about HUGO to make the changes and fix it.. Its very strange.. there is no print directory..

**Comment by MrRiptide** [comment]
I had a similar issue and updating Hugo to 0.79 extended seemed to fix it for me, so it might be worth trying that.

**Comment by gwatts** [comment]
Digging into this further, I reproduced it myself with Hugo 0.74.3, but it was fine with 0.75.0 and later

Wasn't imemdiately obvious why that would be, but I did find a bug in the way i tested for the template's existance, and then loaded it afterwards - It's unclear to me why it worked on 0.75.0 or later at all!

In any case, I have opened a pull request on Docsy to fix it properly as google/docsy#394

In the interim, you have a few options:
*  Upgrade to 0.75.0 of Hugo
* Disable print support (comment out the `[outputs]` section in your `config.toml` file)
* Checkout my fork of Docsy linked from the PR, which includes the fixes

Hope that helps; sorry for the bug!

**Comment by kdamianakis** [comment]
Many thanks for th einformation - I have managed to upgrade the HUGO version and all seems to work ok at the moment.
